src/Core/View.js

```diff
@@ -188,14 +188,8 @@ class View extends THREE.EventDispatcher {
 
         this.addEventListener(VIEW_EVENTS.LAYERS_INITIALIZED, fn);
 
-        this._fullSizeDepthBuffer = null;
-
-        this.addFrameRequester(MAIN_LOOP_EVENTS.BEFORE_RENDER, () => {
-            if (this._fullSizeDepthBuffer != null && this._fullSizeDepthBuffer.needsUpdate) {
-                // clean depth buffer
-                this._fullSizeDepthBuffer = null;
-            }
-        });
+        this._fullSizeDepthBuffer = new Uint8Array(4 * this.camera.width * this.camera.height);
```

Comment:
In `notifyChange` you can remove the check on this variable.

Reply:
yes, good catch!

Reply:
removed